meetings/2021-07/july-15.md

```diff
@@ -0,0 +1,734 @@
+
+MM: For sure, I would solve it better. 
+
+JHD: Yeah, I would agree with that, the Yes. If so then what are, what is the suggestion based on that?
+
+MM:  We've identified a need with some concrete ideas of possible API approaches for satisfying the need, I think we've got something that qualifies very well for a new stage one proposal. I think we should address that need there and we should realize that since the need can be addressed and that there's nothing about the realm proposal in its current state that impedes that in any way that we should allow the Realms proposal in its current state to proceed to stage 3.
+
+JHD: so, it cues get accused empty. So I think that seems a reasonable path forward under one - "condition" is the wrong word because that implies leverage but under the one, like in one circumstance, which is that the "direct object access for Realms" path. It does not appear to me that there will ever that will ever be allowed unless the current membership of TC39 completely turns over are so. It's, it would be kind of weird to ask for consensus on stage for stage one on a proposal or a problem that nothing was written up and put on the agenda before. right now. So it'd be nice if that could be granted, But that alone isn't sufficient because the real piece is, I would kind of need to know if any like, in the same way as when realms, Went to stage two. There was, it was, I think clearly telegraphed that there were these potential blocking concerns for the direct object access approach, right? Is there anyone in the room who has any reason to believe that such a proposal modulo, you know, dealing with various sorts of concerns that proposals run across, but that such a proposal would, would not have a path toward stage 4. Does that make sense? What I'm asking, I'm trying to figure out if it's worth me investing my time. 
+
+MM: Write the proposal, you're referring to is get in transits proposal. We Sure, yes. Okay, Good question.
+
+USA: Is it something that you'd like a temperature check on. 
+
+JHD: I mean, a temperature check seems nice, I think specifically if it would, even though Everyone is always allowed at any time to say, I have an objection or I have a constraint, even though I never mentioned it before, it would feel really bad if I put in time on a proposal like this and then came back in six months, a year, two months, whatever, and was surprised with a sudden constraint, it's happened to me once before it is not a fun thing to do and it's very demotivating. So I really just want to make sure that if there's any possibility that any one, could guess that, that might happen that I could get some hints about it now so that I know whether it's worth putting in my time, this is not really a technical or procedural question, I'm sort of asking you a personal question in somebody. Make sure that I'm not going to about to waste my time.
```

Comment:
```suggestion
JHD: I mean, a temperature check seems nice, I think specifically if it would, even though everyone is always allowed at any time to say, "I have an objection" or "I have a constraint, even though I never mentioned it before", it would feel really bad if I put in time on a proposal like this and then came back in six months, a year, two months, whatever, and was surprised with a sudden constraint. It's happened to me once before, and it is not a fun thing to do and it's very demotivating. So I really just want to make sure that if there's any possibility that any one, could guess that, that might happen that I could get some hints about it now so that I know whether it's worth putting in my time, this is not really a technical or procedural question, I'm sort of asking you a personal question to try to make sure that I'm not about to waste my time.
```